operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -400,20 +419,31 @@ Please refer [iSCSI Metrics](https://github.com/Cray-HPE/sbps-marshal/blob/main/
 
 ## Steps to switch from DVS to iSCSI
 
+This involves this sequence of operations:
+
 * Disable DVS
 * Uninstall CPS
 * Enable iSCSI SBPS
 
-To disable DVS please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/DVS_and_SBPS.md>
+To perform these operations, please refer the section `DVS and SBPS` documented in the publication:
+`HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)`.
+
+## Steps to continue using DVS based projection
 
-To uninstall CPS services please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/Uninstall_CPS.md>
```

Comment:
There used to be 3 references to USS doc sections, but now there is only 1.  But the lead-in text still refers to 3 steps that need to be taken.  Were the other 2 sections completely removed by accident when they should have been converted from links to section names?  I am referring to the "Uninstall CPS" and "Enable iSCSI SBPS" sections.

Reply:
As we discussed, I have addressed this.

Reply:
@deantroe , Please check the addressed text now.

Reply:
I left one new comment to remove an extra word.  Once that is addressed, I'll approve.